modules/controller/rpc-controller.js

```diff
@@ -91,36 +98,41 @@ class RpcController {
 
     async initializeErrorMiddleware() {
         await this.app.use((error, req, res, next) => {
-            let code; let message;
+            let code;
+            let message;
             if (error && error.code) {
                 switch (error.code) {
-                case 400:
-                    code = 400;
-                    message = `Bad request. ${error.message}`;
-                    break;
-                default:
-                    return next(error);
+                    case 400:
+                        code = 400;
+                        message = `Bad request. ${error.message}`;
+                        break;
+                    default:
+                        return next(error);
                 }
                 this.logger.error({ msg: message, Event_name: constants.ERROR_TYPE.API_ERROR_400 });
                 return res.status(code).send(message);
             }
             return next(error);
         });
 
-        this.app.use((error, req, res, next) => {
+        this.app.use((error, req, res) => {
             this.logger.error({ msg: error, Event_name: constants.ERROR_TYPE.API_ERROR_500 });
             return res.status(500).send(error);
         });
     }
 
     enableSSL() {
-        this.sslEnabled = fs.existsSync('/root/certs/privkey.pem') && fs.existsSync('/root/certs/fullchain.pem');
+        this.sslEnabled =
+            fs.existsSync('/root/certs/privkey.pem') && fs.existsSync('/root/certs/fullchain.pem');
```

Comment:
Should we move this to the constants or maybe to the config?

Reply:
I agree, but this is still old controller, for the new one we will move it to config probably. These lines are not changed they are just formated